Sources/PandaSDK/Views/WebViewController.swift

```diff
@@ -217,6 +226,14 @@ final class WebViewController: UIViewController, WKScriptMessageHandler {
             if viewModel?.screenData.id.string == "5077a1da-092c-4ab8-a41f-0a3966a4b326" {
                 viewModel?.onSupportUkraineAnyButtonTap?()
             }
+            if viewModel?.screenData.id.string == "b7627ad6-b5b2-4255-afe6-71842b1f46ec" {
+                if let destination = data["destination"]  {
```

Comment:
I think guard here is not needed because we still need to execute function until the end.